tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -330,9 +332,10 @@ def test_basic_model_monitoring(self, with_sql_target: bool) -> None:
             )
             sleep(choice([0.01, 0.04]))
 
-        sleep(5)
+        sleep(15)
         endpoints_list = mlrun.get_run_db().list_model_endpoints(
```

Comment:
add metrics to `project. list_model_endpoints` so we will be able to use it here.